### PR TITLE
92 refactor popups

### LINKED ISSUE #92: Refactor Popup methods

### PR DESCRIPTION
**Pull request by thervh70**
Will close #92

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/AbstractPopup.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+public abstract class AbstractPopup extends Popup {
+
+	/** Width of the PopUpMenu. */
+	public static final double WIDTH = 260;
+	/** Height of the PopUpMenu. */
+	public static final double HEIGHT = 320;
+	/** Size of the HBox. */
+	protected static final double HBOX_SIZE = 10;
+	/** Size of the VBox. */
+	protected static final double VBOX_SIZE = 50;
```
Comment (thervh70) [inline src/main/java/nl/tudelft/ti2206/group9/gui/AbstractPopup.java]: Technically this is not the size of the box, rather the spacing between the elements.

< naming

File: src/main/java/nl/tudelft/ti2206/group9/gui/AbstractPopup.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        final HBox hbox = new HBox(HBOX_SIZE, left, right);
+        hbox.setAlignment(Pos.CENTER);
+
+        final Node[] content = createContent();
+        final Node[] content2 = Arrays.copyOf(content, content.length + 1);
+        content2[content.length] = hbox;
+        final VBox vbox = new VBox(VBOX_SIZE, content2);
+        vbox.setAlignment(Pos.CENTER);
+
+        getContent().addAll(rect, vbox);
```
Comment (thervh70) [inline src/main/java/nl/tudelft/ti2206/group9/gui/AbstractPopup.java]: The indent does not seem right here. Please review the layout.

< layout

**Comment by thervh70** [comment]
Multiple methods feature indent inconsistency. One of them is mentioned in the comment on the diff stated above. Please fix this.

You really need to merge master into this branch and iron out all the errors and warnings before we can close this PR. If you have succesfully done that, this branch can be merged!

< review, layout

**Comment by MKerkhof** [comment]
+1 on Robin's comment. There are still some warnings left, can be merged after resolving them.

<review

**Comment by mpsijm** [comment]
Thanks for your comments, I have processed them. Then next reviewer may merge this in my opinion! :)
<review,layout,naming

**Comment by MathiasMeuleman** [comment]
Just finished reviewing, looks fine! 
Will merge :)

<review

**Pull request merged**